src/components/content/CookieBar.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react'
+import styled from 'styled-components'
+import { useCookies } from 'react-cookie'
+
+export default function CookieBar() {
+  const [cookies, setCookie] = useCookies(['consent'])
+  let showCookieBar = cookies.consent
+
+  function accept() {
+    const newDate = new Date()
+    const day = newDate.getDate()
+    const month = newDate.getMonth() + 1
+    const year = newDate.getFullYear()
+
+    setCookie(
+      'consent',
+      `${year}-${month < 10 ? `0${month}` : `${month}`}-${day}`
+    )
+  }
+
+  if (!showCookieBar) return null
```

Comment:
That doesn't match the current behavior of the consent banner. What we do currently:

1. Fetch the current revision of the privacy policy via https://de.serlo.org/privacy/json
2. Check whether the consented revision matches the current revision (if any)
3. If not, we show the cookie banner
4. On consent, we persist the revision the user has consented to. Furthermore, we track that via Google Analytics to get a feeling how many people ignore the consent banner (not necessary as it depends on #64)

Please have a look at the current implementation at https://github.com/serlo/serlo.org/blob/master/packages/public/client/src/frontend/modules/consent-banner.ts#L36